src/controllers/managing.officer.corporate.controller.ts

```diff
@@ -0,0 +1,8 @@
+import { Request, Response } from "express";
+import * as config from "../config";
+import { logger } from "../utils/logger";
+
+export const get = (req: Request, res: Response) => {
+  logger.debug(`GET ${config.MANAGING_OFFICER_CORPORATE_PAGE}`);
+  return res.render(config.MANAGING_OFFICER_CORPORATE_PAGE);
```

Comment:
Do you know the page that comes before this right now? Will need to pass in the backLinkUrl if so